bot/persistence/resources/init_db.sql

```diff
@@ -3,4 +3,9 @@ CREATE TABLE IF NOT EXISTS Modmail(
     Author TEXT NOT NULL,
     StatusID SMALLINT NOT NULL DEFAULT 1,
     Timestamp TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS BotOnly(
+    Channel TEXT PRIMARY KEY,
```

Comment:
No Channel Name. Easier human readable and easiert to use in code.

Reply:
So you use the channels name to check if it is in bot-only mode? That's not a good idea in my opinion because the name of a channel isn't unique and can change over time in comparison to the channel ID.